Frontend/components/orders/ordersForm.js

```diff
@@ -82,22 +68,18 @@ const OrdersForm = () => {
             setShipToError(true);
         }
 
-        if (quantity === "") {
-            setQuantityError(true);
-        }
         if (invoiceAmount === "") {
             setInvoiceAmountError(true);
         }
         // request is sent after all input are valid
-        if (customerID && customerName && productName && date && shipTo && quantity && invoiceAmount) {
+        if (customerID && customerName  && date && shipTo &&  invoiceAmount) {
```

Comment:
Could you format it evenly? :)

Reply:
done with the commit "evenly formatted" 👍